apps/namadillo/src/App/Layout/Navigation.tsx

```diff
@@ -57,14 +65,53 @@ export const Navigation = (): JSX.Element => {
   return (
     <div className="min-h-full flex flex-col justify-between gap-10 p-6 pb-8">
       <ul className="flex flex-col gap-4">
-        {menuItems.map((item) => (
-          <li key={item.label}>
-            <SidebarMenuItem url={item.url}>
-              {item.icon}
-              {item.label}
-            </SidebarMenuItem>
-          </li>
-        ))}
+        {menuItems.map((item) => {
+          const shieldingRoute = item.label === "Shield";
+
+          const highlightShieldItem =
+            item.label === "Shield" &&
+            ([routes.maspShield, routes.ibc] as string[]).includes(
+              location.pathname
+            );
+
+          const highlightTransferItem =
+            item.label === "Transfer" &&
+            (
+              [
+                routes.maspUnshield,
+                routes.ibcWithdraw,
```

Comment:
Are some of those routes still in use? If we do not need them then maybe it's better to get rid off them for simplicity. Unless we have to support some old urls.